cloud-infrastructure/storage/block-storage/asset/block-volume-backup-and-restore.txt

```diff
@@ -0,0 +1,155 @@
+License
```

Comment:
Should this be run as a shell script? If so, start start with `#!/bin/bash`

Reply:
no, the idea is to copy and paste code sections into a terminal